middle_end/flambda2/types/grammar/type_grammar.ml

```diff
@@ -52,6 +52,11 @@ type t =
   | Region of head_of_kind_region TD.t
 
 and head_of_kind_value =
+  { non_null : head_of_kind_value_non_null;
+    is_null : bool Or_unknown.t (* CR vlaviron: define a dedicated type *)
```

Comment:
It might be the right time to fix this CR

Reply:
Done